app/models/ect_at_school_period.rb

```diff
@@ -17,7 +11,7 @@ class ECTAtSchoolPeriod < ApplicationRecord
   # Associations
   belongs_to :school, inverse_of: :ect_at_school_periods
   belongs_to :teacher, inverse_of: :ect_at_school_periods
-  belongs_to :appropriate_body
+  belongs_to :school_reported_appropriate_body, class_name: 'AppropriateBody', optional: true
```

Comment:
We have `belongs_to_required_by_default = false` [set for the app](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/config/application.rb#L20) so we can skip the optional here. It's set that way so Rails doesn't emit errors on the relationship rather than on the foreign key (which means error summary links don't work)